plugins/modules/dcnm_network.py

```diff
@@ -92,6 +92,55 @@
         description: 'Gateway with subnet for the network'
         type: ipv4
         required: false
+      is_l2only:
+        description: 'Layer 2 only network'
+        type: bool
+        required: false
+        note: If specified as true, VRF Name(vrf_name) should be specified as "NA"
+        default: false
+      vlan_name:
+        description: 'Name of the vlan configured'
+        type: str
+        note: if > 32 chars enable, system vlan long-name on switch
```

Comment:
In DCNM GUI, when we specify vlan_name greater than 32 characters without configuring "system vlan long-name" command, the fabric goes to "out-of-sync" state on deploying. This will be the case with ansible network module too.